install_latest_gauge_nightly.ps1

```diff
@@ -7,12 +7,15 @@ else {
     $bit="x86"
 }
 
-$gauge_file_name="gauge-$($gauge_latest_nightly_version)-windows.$($bit).exe"
+$gauge_exe_folder_name="gaugeExe"
+$gauge_file_name="$gauge_exe_folder_name\\gauge.exe"
+$gauge_zip_name="gauge-$($gauge_latest_nightly_version)-windows.$($bit).zip"
 
-Invoke-WebRequest -Uri "https://bintray.com/gauge/Gauge/download_file?file_path=windows%2F$gauge_file_name" -OutFile $gauge_file_name
+Invoke-WebRequest -Uri "https://bintray.com/gauge/Gauge/download_file?file_path=windows%2F$gauge_zip_name" -OutFile $gauge_zip_name
+Expand-Archive $gauge_zip_name -DestinationPath $gauge_exe_folder_name
 
 Write-Host "Installing $($gauge_latest_nightly_version) from $($gauge_file_name)..."
 
-& ".\$($gauge_file_name)" /S
+& ".\$gauge_file_name" /S
```

Comment:
This is not required. Earlier the `exe` was the installer which was launched with a `/S` flag. Now since we are expanding archive this is a redundant step.